src/ripple/app/main/DBInit.cpp

```diff
@@ -90,14 +90,29 @@ const char* LedgerDBInit[] =
     );",
     "CREATE INDEX IF NOT EXISTS SeqLedger ON Ledgers(LedgerSeq);",
 
+    // InitialSeq field is the current ledger seq when the row
+    // is inserted. Only relevant during online delete
     "CREATE TABLE IF NOT EXISTS Validations   (                   \
+        LedgerSeq   BIGINT UNSIGNED,                \
+        InitialSeq  BIGINT UNSIGNED,                \
         LedgerHash  CHARACTER(64),                  \
         NodePubKey  CHARACTER(56),                  \
         SignTime    BIGINT UNSIGNED,                \
         RawData     BLOB                            \
     );",
+    // This will error out if the column already exists,
+    //  but DatabaseCon intentionally ignores errors.
+    "ALTER TABLE Validations                        \
+        ADD COLUMN LedgerSeq       BIGINT UNSIGNED;",
+    "ALTER TABLE Validations                        \
+        ADD COLUMN InitialSeq      BIGINT UNSIGNED;",
+
```

Comment:
Unless I'm missing something, we used to do this all the time when we had database changes. We'd query the structure of the database and if it was missing a column we needed, we'd add that column.